etor/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/UpdateUniversalServiceIdentifierTest.groovy

```diff
@@ -0,0 +1,203 @@
+package gov.hhs.cdc.trustedintermediary.etor.ruleengine.transformation.custom
+
+import gov.hhs.cdc.trustedintermediary.ExamplesHelper
+import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
+import gov.hhs.cdc.trustedintermediary.external.hapi.HapiHelper
+import org.hl7.fhir.r4.model.Bundle
+import org.hl7.fhir.r4.model.ServiceRequest
+import spock.lang.Specification
+
+class UpdateUniversalServiceIdentifierTest extends Specification {
+    def transformClass
+    def fhirResource
+    def args = Map.of(
+    "checkValue", "54089-8",
+    "codingSystem", "LN",
+    "alternateId", "CDPHGSPEAP"
+    )
+
+    def setup() {
+        TestApplicationContext.reset()
+        TestApplicationContext.init()
+        TestApplicationContext.injectRegisteredImplementations()
+
+        transformClass = new UpdateUniversalServiceIdentifier()
+        fhirResource = ExamplesHelper.getExampleFhirResource("../Test/Results/005_AL_ORU_R01_NBS_Simplified_1_hl7_translation.fhir")
+    }
+
+    def "skip transformation if the coding identifier is missing"() {
+        given:
+        def bundle = fhirResource.getUnderlyingResource() as Bundle
```

Comment:
Shouldn't the first part of this test call the transform method with an empty map for the args? Because as it's written it doesn't actually call any mutating code

Reply:
Nevermind, I misread the test, resolving!